build.gradle

```diff
@@ -180,19 +182,17 @@ task pushDockerImage {
 
 task removeOldDockerContainer(type: Exec) {
     commandLine "bash", "-c",
-            "for DC in `docker ps -f 'name=$dockerContainerName' | grep -o $dockerContainerName`; do docker stop \$DC; done && " +
-            "for DC in `docker ps -a -f 'name=$dockerContainerName' | grep -o $dockerContainerName`; do docker rm \$DC; done"
+            "docker ps -aqf name=$dockerContainerName | xargs -r docker rm -f"
 }
 
 task startDockerContainer(type: Exec) {
     commandLine "bash", "-c", "docker run -e \"NAKADI_OAUTH2_ENABLED=false\" " +
-            "--net container:local-kafka --name $dockerContainerName -i -t -d " +
-            "\$(docker images | grep -m 1 \"^$dockerGroup/$dockerApplicationName\" | awk \'{print \$3}\')"
-    dependsOn buildDockerImage, removeOldDockerContainer, startKafkaInDockerBound
+            "--net container:$storagesContainerName --name $dockerContainerName -i -t -d $dockerImageTag"
+    dependsOn buildDockerImage, removeOldDockerContainer, startStoragesInDockerBound
 }
 
 task stopDockerContainer(type: Exec) {
-    commandLine "bash", "-c", "docker stop \$(docker ps | grep \$(docker images | grep -m 1 \"^$dockerGroup/$dockerApplicationName\" | awk '{print \$3}') | awk '{print \$1}')"
+    commandLine "bash", "-c", "docker ps -qf name=$dockerContainerName | xargs -r docker rm -f"
```

Comment:
Besides stopping the container it also removes it. Maybe it would be nice to change the name of the task to stopAndRemoveDockerContainer?